index.bs

```diff
@@ -1428,6 +1428,7 @@ To <dfn>parse filter values</dfn> given a |value|:
 1. If |value| is not a [=map=], return null.
 1. Let |result| be a new [=filter map=].
 1. [=map/iterate|For each=] |filter| → |data| of |value|:
+    1. If |filter| starts with "`_`", return null.
```

Comment:
Is there a more precise algorithm we can use here, similar to https://wicg.github.io/attribution-reporting-api/#parse-an-aggregation-key-piece ?

Reply:
Updated to use: https://infra.spec.whatwg.org/#string-starts-with